doc/_static/custom_math/net_import_share.yaml

```diff
@@ -0,0 +1,44 @@
+# Force upper limit on carrier imports within nodes or across all nodes as a share of all carrier flows at a node/across nodes.
+# Imports of transmission technologies are defined by `flow_out` (outflow from a transmission technology that originated in a remote node).
+
+constraints:
+  net_import_share_max:
+    description: Limit upper bound on electricity imports within nodes as a share of all electricity flows at each node.
+    foreach: [nodes, timesteps]
+    where: "defined(techs=test_transmission_elec, within=nodes, how=any)"
+    equations:
+      - expression: net_import_share * sum(flow_out[techs=$transmission_techs, carriers=electricity], over=techs) <= $total_energy_balance
+    sub_expressions:
+      total_energy_balance:
+        - expression: sum(flow_out[carriers=electricity], over=techs) - sum(flow_in[carriers=electricity], over=techs)
+    slices:
+      transmission_techs:
+        - expression: get_transmission_techs(test_transmission_elec)
```

Comment:
Once helper functions like `get_transmission_techs` are documented elsewhere, those docs could be linked to from here

Reply:
They're already documented in the [API docs](https://calliope.readthedocs.io/en/latest/api/api.html#module-calliope.backend.helper_functions), although that is quite messy so I was planning to also give a run-down of them in the math section I'm writing in #423.